src/rust/lib/graph-editor/src/lib.rs

```diff
@@ -1440,6 +1445,7 @@ impl application::shortcut::DefaultShortcutProvider for GraphEditor {
              , Self::self_shortcut(shortcut::Action::press        (&[Key::Character("d".into())])               , "set_test_visualization_data_for_selected_node")
              , Self::self_shortcut(shortcut::Action::press        (&[Key::Character("f".into())])               , "cycle_visualization_for_selected_node")
              , Self::self_shortcut(shortcut::Action::release      (&[Key::Control,Key::Enter])                  , "enter_selected_node")
+             , Self::self_shortcut(shortcut::Action::release      (&[Key::Control,Key::ArrowUp])                , "step_out_of_node")
```

Comment:
write todo that this needs to change. In the future, enter should enter nodes, while cmd / ctrl + enter should exit them. (the same should happen after double press in background).

Reply:
Will create an issue.

Reply:
https://github.com/enso-org/ide/issues/628